packages/block-editor/README.md

```diff
@@ -430,6 +430,19 @@ _Returns_
 
 -   `string|null`: A font-size value using clamp().
 
+### getCustomValueFromPreset
```

Comment:
I totally forgot that we're of course working in a different package when making changes to the Spacer block, so this function needs to be exported now.

Shall we rename it to `getSpacingCustomValueFromPreset` or `getCustomSpacingValueFromPreset` since it's being exported from the block editor package? That way if we have other functions for different kinds of presets the need to be exported in the future, we won't have a collision?

That said, from a quick search I see that the native version of `index.native.js` for the block editor package is already exporting this function as-is here: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/index.native.js#L79.

So, perhaps another option would be to leave it as-is and if in the future we need a more generic `getCustomValueFromPreset` to be exported that handles other kinds of presets, then we could add a parent function that internally calls the existing spacing one.

What do you think?

Reply:
I'd leave it as is for now, and then perhaps if/when we need to handle other types of presets, we can reimplement this function to deal with any preset, so we don't have to change existing usage (we'd probably want to move it out of the spacing utils if we did that of course).